docs/source/platform/operation-registry.md

```diff
@@ -146,14 +143,14 @@ const server = new ApolloServer({
   // ...
   // New configuration
   plugins: [
-    require('apollo-server-plugin-operation-registry')({
-      forbidUnregisteredOperations: true,
-    }),
-  ],
+    require("apollo-server-plugin-operation-registry")({
```

Comment:
If this is a result of our loose policy to _Prettier_ all the things before putting them in docs, great!  If not, I still support it! :)